student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```

Comment:
- [ ] In your `student.rb`, you implemented the `Teacher` class. Kindly implement both the `Teacher` and `Student` classes in their respective files.